CHANGELOG.md

```diff
@@ -83,7 +84,9 @@ By default its true.
 
 ## 0.1.3
 
-* added support for `sql.js`. To use it you just need to install `npm i sql.js` and use `sqljs` as driver type ([#894](https://github.com/typeorm/typeorm/pull/894)).
+* added support for `
+
+`. To use it you just need to install `npm i sql.js` and use `sqljs` as driver type ([#894](https://github.com/typeorm/typeorm/pull/894)).
```

Comment:
What happened here?

Reply:
What the hell? I used Github's web editor, no idea where this is coming from.